test/runnable/cstuff2.c

```diff
@@ -0,0 +1,22 @@
+
+int printf(const char *, ...);
+void exit(int);
+
+void test1()
+{
+    static int a[3] = {1, 2, 3};
+    if (a[0] != 1 ||
+	a[1] != 2 ||
+	a[2] != 3)
```

Comment:
I know its only a test, but really? The indenting is horrendous.

Reply:
forgot to detab it